booking/src/main/java/org/dcsa/conformance/standards/booking/checks/BookingChecks.java

```diff
@@ -650,10 +668,14 @@ public static ActionCheck responseContentChecks(UUID matched,String standardsVer
       bookingStatus.wireName()
     ));
     checks.addAll(STATIC_BOOKING_CHECKS);
-    if (standardsVersion.equals("2.0.0-beta-1")) {
+    if (standardsVersion.equals("2.0.0-Beta-1")) {
       checks.add(IS_EXPORT_DECLARATION_REFERENCE_PRESENT);
       checks.add(IS_IMPORT_DECLARATION_REFERENCE_PRESENT);
     }
+    if (standardsVersion.equals("2.0.0-Beta-2")) {
+      checks.add(IS_EXPORT_DECLARATION_REFERENCE_ABSENCE);
+      checks.add(IS_IMPORT_DECLARATION_REFERENCE_ABSENCE);
+    }
```

Comment:
I would go with a `!standardsVersions.equals("2.0.0-Beta-1")` here since we assume this will apply to all future versions (until further notice).

The alternative if we have to remember to update the `if` with every beta from here on out, which we will likely forget and suddenly this check becomes a `Beta-2`-only check.

Reply:
Good one Niels.. I agree with you